lib/GADS/Datum/Person.pm

```diff
@@ -265,11 +267,9 @@ sub html
         $t = encode_entities $t;
         push @details, $site->register_freetext2_name.": $t";
     }
-    my $details = join '<br>', map {encode_entities $_} @details;
+    my $details = join '<br>', @details;
```

Comment:
Can this whole html() function be removed now? At this point, $details is part HTML and part not-HTML, so think it would be better removed altogether than accidentally used.